webapp/stores/team_store.jsx

```diff
@@ -148,6 +148,21 @@ class TeamStoreClass extends EventEmitter {
     getTeamListings() {
         return this.teamListings;
     }
+
+    isTeamAdminForCurrentTeam() {
```

Comment:
good catch...changing this slightly.  shouldn't require userId since this list is only TeamMember objects for the current user.